interface/weno/indexrx.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ *  @package OpenEMR
+ *  @link    http://www.open-emr.org
+ *  @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @copyright Copyright (c) 2020 Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once("../globals.php");
+require_once("logsync.php");
+
+use OpenEMR\Rx\Weno\Container;
+
```

Comment:
will need an acl check here (depends on what decide to use on the menu item link)